Package.swift

```diff
@@ -12,6 +12,7 @@ let hostingUrl = "https://amazon-chime-sdk-ios.s3.amazonaws.com"
 // Checksums for SDKs to verify that the hosted archive file matches the archive declared in the manifest file.
 let SDKChecksum = "bc4193855ec5b96732846b27954573ee4b9ec497f930e9a678125ee5b3a8f49a"
 let SDKMediaChecksum = "03751799d43c5325524f9056057530f25da4ccdc6d712acb24f556471f0e939a"
+let SDKMediaNoVideoCodecsChecksum = "03751799d43c5325524f9056057530f25da4ccdc6d712acb24f556471f0e939a"
```

Comment:
This value is randomly set for now and will be updated once we run the sdk_spm_version_update script

Reply:
can we run script first?

Reply:
I'm thinking of if we should set it to real value.